common/sw-update/dev-updates-profile.cpp

```diff
@@ -47,7 +47,8 @@ namespace rs2
                 version& current_version((comp == FIRMWARE) ? _update_profile.firmware_version : _update_profile.software_version);
                 {
                     version_info experimental_update;
-                    if (try_parse_update(_versions_db, _update_profile.device_name, EXPERIMENTAL, comp, experimental_update))
+                    auto parse_update_stts = try_parse_update(_versions_db, _update_profile.device_name, EXPERIMENTAL, comp, experimental_update);
+                    if ( parse_update_stts )
```

Comment:
Isn't this now a status enum? You're treating it like an enum, looks like a possible bug?

Reply:
Fixed